quic/s2n-quic-platform/src/io/tokio/builder.rs

```diff
@@ -122,6 +122,17 @@ impl Builder {
         Ok(self)
     }
 
+    /// Configures Generic Segmentation Offload (GSO)
+    ///
+    /// By default, GSO will be used unless the platform does not support it or an attempt to use
+    /// GSO fails. If it is known that GSO is not available, set this option to explicitly disable it.
+    pub fn with_gso(mut self, enabled: bool) -> io::Result<Self> {
+        if !enabled {
+            self.max_segments = 1.try_into().expect("1 is always a valid MaxSegments value");
+        }
```

Comment:
if we're keeping the old one around, would it be better to just do:
```suggestion
        if !enabled {
            return self.with_gso_disabled()
        }
```

Reply:
Yeah good call